index.md

```diff
@@ -219,7 +219,7 @@ You might need records if you later decide to [take further action](#action). In
 
 ### Getting a Response {#response}
 
-You might get a quick response, or it might take longer to hear back. It depends on the organization's culture, policies, and internal systems.
+You might get a quick response, or it might take longer to hear back. It depends on the organization's culture, policies, and internal systems. Government agencies and other organizations providing public services may be required to respond.
```

Comment:
I think that last sentence was an intentional hint without getting into the weeds of legal advice.